doc/encryption.md

```diff
@@ -146,7 +146,7 @@ The exception has the form:
      Caused by: java.lang.RuntimeException: Unexpected type: 5
             at com.cloudant.sync.datastore.BasicDatastore.getFullRevisionFromCurrentCursor(BasicDatastore.java:1709)
             at com.cloudant.sync.datastore.BasicDatastore.getDocumentInQueue(BasicDatastore.java:284)
-            at com.cloudant.sync.datastore.BasicDatastore.createDocument(BasicDatastore.java:681)
+            at com.cloudant.sync.datastore.BasicDatastore.createJsonDocument(BasicDatastore.java:681)
```

Comment:
Well I was being semi-serious. The stacktrace was correct at the time of writing. If your'e going to update the method name then you should ensure that the entire stacktrace is correct. I say you might as well leave it how it was.